src/core/auth/dto/register.dto.ts

```diff
@@ -13,6 +13,11 @@ export class RegisterDto {
   @MinLength(1)
   lastname: string;
 
+  @ApiProperty()
+  @IsString()
+  @MinLength(2)
+  document: string;
```

Comment:
What is document xD?

Reply:
cedula en ingles

Reply:
La verdad lo veo como un nombre bien poco descriptivo... pero no se como mas almacenarlo, maybe
```suggestion
  documentCi: string;
```
Si no se nos ocurre algo
```suggestion
  cedula: string; //xd
```
o
```suggestion
  citizenDocument: string;
```
Creo que la mejor seria
```suggestion
  documentNumber: string;
```

Reply:
documentNumber me voy mas por esa, pero investiguen a ver que se hace en esos casos

Reply:
o `identification`, idk

Reply:
En ese mismo orden de ideas, debemos hacer que este campo sea obligatorio, que se ponga en el registro y que además sea único, que te evite el registro si detecta a alguien mas con esa misma cédula